cloud_governance/policy/ec2_idle.py

```diff
@@ -32,15 +40,14 @@ def __organise_instance_data(self, instances_data: list):
         """
         organize_data = []
         for instance in instances_data:
-            instance['LaunchTime'] = str(instance['LaunchTime'])
+            instance['LaunchTime'] = self.__convert_datetime_to_string(instance['LaunchTime'])
```

Comment:
No need method for 1 line

Reply:
Used multiple times. So I created a method